### DISCUSSION TITLE
v0.3 is out for testing

### DISCUSSION THREAD

**Discussion by psanders** [fonoster/yaps#322] [Announcements]
![release_v1](https://user-images.githubusercontent.com/539774/162630724-1583c35d-baf7-42cf-8465-e5a343aae040.png)

# v0.3 is out for testing

We're happy to announce that [Fonoster 0.3 is available for testing](https://github.com/fonoster/fonoster/tree/0.3). Fonoster 0.3 is a feature release that will replace the main branch and includes many improvements across all the components. 

This version ships with support for Secrets, better integration with DialogFlow ES/CX, and significant improvements around observability.

We're grateful for the support from individual contributors helping make this release of Fonoster the best it can be. You can help by raising [issues](http://github.com/fonoster/fonoster/issues), recommending features, sharing Fonoster with your friends, and contributing to the project on GitHub.

> See [here](https://github.com/orgs/fonoster/projects/9) are current roadmap.

**Comment by PeterOsei** [comment]
Can I be added to the console for testing too?!

**Reply by YuriCodes** [reply]
Hello, Peter 👋! You first have to log in to the [console](https://console.fonoster.io/) and get familiar with the dashboard. 
You can find tutorials at https://learn.fonoster.com/ 
And for set up help https://learn.fonoster.com/docs/overview
If you need any more guidance don't forget to join our [discord community](https://discord.gg/mpWSRUhG7e) we have a get-help channel. 

Thank you for your interest in Fonoster.

**Reply by jhoanborges** [reply]
I'm having this error right after github log in
![image](https://user-images.githubusercontent.com/32471957/192582948-b04410b0-58d2-4c3d-b309-6ac8cbf5fc21.png)

**Reply by efraa** [reply]
Hey @jhoanborges 

Thank you for reporting the issue. We triaging and confirming it as an internal bug. We fixed it, try again. :octocat:

**Comment by kic86** [comment]
How do I get access for testing?

**Reply by psanders** [reply]
Hi 👋

Ideally, you first log in to the [console](https://console.fonoster.io) and get familiar with the tool. You will find tutorials at https://learn.fonoster.com for some basic tasks.

Also, we can schedule a Coffee Chat, and I can give a walkthrough of the system and answer some questions.

Here is my [Calendly](https://calendly.com/fonoster-psanders) link.

**Comment by PeterOsei** [comment]
Hi Yuri,

I have already done that.

Can I get access to the api testing

Regards

Regards,

Peter Osei
________________________________

**Reply by psanders** [reply]
@PeterOsei 

Thanks for your interest in Fonoster.

Can you clarify the question? 

We are not sure we understand which API testing you are referring to.

Let us know.
